### ISSUE TITLE
BotAuth is obsolete?

### ISSUE THREAD

**Issue by dnws** [mattdot/botauth#61]
BotBuilder since release **3.15.1.0** supports generic OAuth flows and AADv1/v2 authentication methods out of the box. There are examples for each auth flow in [official documentation](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-tutorial-authentication?view=azure-bot-service-3.0). Integrated flows are also capable of making all the verifications in the background. This project even being presented on Build 2018 seem to be abandoned.

@mattdot should it be discontinued and marked as such?

**Comment by josephktcheung** [comment]
@dnws I want to use built-in OAuth in bot service too but the lack of documentation makes it impossible for me to implement the flow with Auth0. Please read the comments in the documentation page where people are complaining about the lack of documentation, examples and problems like refreshing tokens not working etc.

BotAuth can serve as an alternative to people who are not yet satisfied with the bot service OAuth implementation, until most questions in the official documentation comments section are answered, 
 documentation is more polished and more examples are provided.

**Comment by rgarita** [comment]
I had issues with BotBuilder OAuth cards on teams, so I'm back on this one.......

**Comment by EricDahlvang** [comment]
@rgarita What were the issues you ran into with OAuth cards on teams?

**Comment by belcsy** [comment]
I have some issues with the BotBuilder OAuth card as well. 

The error i constantly get is that 'Parameter "url" must be a string, not undefined' in the Bot Emulator. I have tried to change the 'value: undefined' to a fixed value in OAuthCard.js, but it is not working as well. I couldn't find any documentation on where exactly i should provide a url...

I bounced back to BotAuth, which is now working better for me
